src/main/java/com/example/demo/controllers/MediaController.java

```diff
@@ -0,0 +1,36 @@
+package com.example.demo.controllers;
+
+import com.example.demo.entities.Media;
+import com.example.demo.repositories.MediaRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@CrossOrigin
+public class MediaController {
+
+    @Autowired
+    private MediaRepository mediaRepository;
+
+    //only useful for testing purposes
+    @GetMapping("/media")
+    public List<Media> getAllMedia(){
+        return mediaRepository.findAll();
+    }
+
+    @GetMapping ("/media/{species}")
```

Comment:
Nothing wrong, though as a matter of personal preference, having the endpoint be something along the lines of `/animals/{id}/media` it'd make it a bit more RESTful and predictable.